### PR TITLE
add a legend in the dashboard

### PR DESCRIPTION
**Pull request by nlehuby**
because nobody likes my emojis :crying_cat_face:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,6 +20,32 @@
         </style>
     </head>
     <body>
+        <article>
+            <h2>Cosmogony metrics</h2>
+            <p>You can check below the number of zones by country for each type of zone, and compare this number with the actual number of zones in the real world.
+                <details>
+                    <ul>
+                        <li>✅✅ : ok
+                        <li>❎❎ : ko, too few or too many zones
```
Comment (antoine-de) [inline index.html]: isn't there a red cross ? it's a bit difficult to see the difference with the ticks.

maybe ❌?

File: index.html [hunk 2]
```diff
@@ -20,6 +20,32 @@
...
+                        <li>❎ known failures : we know that somes zones are still missing in OpenStreetMap
```
Comment (antoine-de) [inline index.html]: ✴ ? (but it seems not to be always like this depending on the font)

File: index.html [hunk 3]
```diff
@@ -20,6 +20,32 @@
...
+                        <li>✅😍 : we did think that some zones were missing in OpenStreetMap, but it's actually ok
```
Comment (antoine-de) [inline index.html]: ✌ ?
✅✌ ?

File: index.html [hunk 4]
```diff
@@ -20,6 +20,32 @@
         </style>
     </head>
     <body>
+        <article>
+            <h2>Cosmogony metrics</h2>
+            <p>You can check below the number of zones by country for each type of zone, and compare this number with the actual number of zones in the real world.
```
Comment (antoine-de) [inline index.html]: should we had something to tell how the found the values?  a link to our readme or mention wiikidata ? (but a mention to our readme might be enough)

Comment (nlehuby) [reply index.html]: done

**Comment by nlehuby** [comment]
new icons:

screenshot-2018-5-18 cosmogony volumetric data dashboard

**Comment by antoine-de** [comment]
I close this as it has been integrated in https://github.com/osm-without-borders/cosmogony_explorer/pull/12

**Pull request closed without merging**